server/routes/helpdesk/validators/emailValidator.test.ts

```diff
@@ -1,7 +1,7 @@
 import validateEmail from './emailValidator'
 
 describe('emailValidator', () => {
-  Array.of('user@aardvark.com.cjsm.net', 'mailroom-user@hmpps.gov.uk').forEach(email => {
+  Array.of('user@aardvark.com', 'user@aardvark.com.notcjsm.net', 'mailroom-user@hmpps.gov.uk').forEach(email => {
```

Comment:
`user@aardvark.com.notcjsm.net` - proves the implementation is marginally better than just "endsWith('cjsm.net')"